h5p-pt_PT.po

```diff
@@ -0,0 +1,894 @@
+msgstr "O H5P vai buscar agora tipos de conteúdo directamente do H5P Hub. Para fazer isso, o plug-in H5P irá comunicará com o H5P.org uma vez por dia para ir buscar informações sobre tipos de conteúdo novos e atualizados. Irá enviar dados anónimos ao hub sobre o uso do H5P. Pode desactivar a contribuição de dados e/ou o H5P Hub nas configurações do H5P."
+
+#: admin/class-h5p-plugin-admin.php:338 admin/views/new-content.php:42
+msgid "If you need any help you can always file a <a href=\"%s\" target=\"_blank\">Support Request</a>, check out our <a href=\"%s\" target=\"_blank\">Forum</a> or join the conversation in the <a href=\"%s\" target=\"_blank\">H5P Community Chat</a>."
+msgstr "Se necessita de ajuda pode sempre requerer<a href=\"%s\" target=\"em branco\">Solicitação de suporte</a>, verifique o nosso <a href=\"%s\" target=\"_blank\">Forum</a> junte-se na conversa no <a href=\"%s\" target=\"_blank\"> Chat da Comunidade H5P</a>."
+
+#: admin/class-h5p-plugin-admin.php:396 public/class-h5p-plugin.php:753
+msgid "H5P Content"
+msgstr "Guia do autor do conteúdo H5P"
+
+#: admin/class-h5p-plugin-admin.php:399
+msgid "All H5P Content"
+msgstr "Todo o guia do autor do conteúdo H5P"
+
+#: admin/class-h5p-plugin-admin.php:399
+msgid "My H5P Content"
+msgstr "Meu guia do autor do conteúdo H5P"
+
+#: admin/class-h5p-plugin-admin.php:402
+msgid "Add New"
+msgstr "Adicionar novo"
+
+#: admin/class-h5p-plugin-admin.php:408
+msgid "Libraries"
+msgstr "Bibliotecas"
+
+#: admin/class-h5p-plugin-admin.php:415 admin/views/my-results.php:14
+msgid "My Results"
+msgstr "Meus resultados"
+
+#: admin/class-h5p-plugin-admin.php:592
+msgid "You're trying to upload content of an older version of H5P. Please upgrade the content on the server it originated from and try to upload again or turn on the H5P Hub to have this server upgrade it for your automaticall."
+msgstr "Está a tentar carregar guia do autor do conteúdo de uma versão antiga do H5P. Por favor, atualize o guia do autor do conteúdo no servidor original e tente carregar novamente ou active o H5P Hub para ter este servidor a atualizar automaticamente."
+
+#: admin/class-h5p-plugin-admin.php:749
+msgid "Invalid content"
+msgstr "Guia do autor do conteúdo inválido"
+
+#: admin/class-h5p-plugin-admin.php:753 admin/class-h5p-plugin-admin.php:1186
+msgid "Invalid security token"
+msgstr "Token de segurança inválido"
+
+#: admin/class-h5p-plugin-admin.php:982
+msgid "Loading data."
+msgstr "Carregando dados."
+
+#: admin/class-h5p-plugin-admin.php:983
+msgid "Failed to load data."
+msgstr "Falha ao carregar dados."
+
+#: admin/class-h5p-plugin-admin.php:984
+msgid "There's no data available that matches your criteria."
+msgstr "Não existe conteúdo disponível que preencha os seus critérios."
+
+#: admin/class-h5p-plugin-admin.php:985
+msgid "Page $current of $total"
+msgstr "Página $atual de $total"
```

Comment:
There's no diff here, but line 408 seems to have been the only change. Changed from
`Page $current of $total"` to `"Página $atual de $total"` which doesn't seem to make sense, assuming that `$current` is a placeholder.